os/common/ports/ARMCMx/compilers/GCC/utils/fault_handlers_v7m.c

```diff
@@ -133,12 +133,12 @@ static void _init_fault_info(void) {
 static void _save_fault_info(void) {
 	memset(&fault_info, 0, sizeof(fault_info));
 
-	if (ch.rlist.current) {
+	if (currcore->rlist.current) {
```

Comment:
currcore-> points to the current core.

![image](https://user-images.githubusercontent.com/23428162/147865022-4875219e-f32d-43e1-9c1e-c8bea1d60403.png)